task-07/README.md

```diff
@@ -21,13 +21,22 @@ Note: on MAC use `bridge0` instead of `tapbr0`.
 
 ## Task 7.2: Extend `gnrc_minimal` application
 * Add the `gnrc_udp` module to the application's [Makefile](Makefile)
+* To be able to receive packets, a packet buffer must be created using [msg_init_queue](https://doc.riot-os.org/group__core__msg.html#ga480e6f32c8ab18579b62a890f3fda2cd):
```

Comment:
That is not entirely correct. The `netreg ` (which is mentioned below) requires a queue in the default case (check description for [`gnrc_netreg_register()`](https://doc.riot-os.org/group__net__gnrc__netreg.html). This is also stated [here](https://doc.riot-os.org/group__net__gnrc.html) under "Architecture" point 1. The packet buffer has nothing to to with that. Please rephrase and reference one of the sources I linked (if you think it makes sense). Now that this is a requirement of `netreg`, wouldn't it make sense to explain it after the function is mentioned the first time (see some lines below)?

Reply:
Furthermore, please reduce the line width to something around 80-90.